src/main/java/net/pms/formats/v2/AudioProperties.java

```diff
@@ -219,4 +219,15 @@ public static int getSampleFrequencyFromLibMediaInfo(String mediaInfoValue) {
 			return result;
 		}
 	}
+
+	@Override
+	public String toString() {
+		StringBuilder result = new StringBuilder();
+		result.append("Channel(s): ").append(getNumberOfChannels());
+		if (getAudioDelay() != 0) {
+			result.append(", Delay: ").append(getAudioDelay());
+		}
+
```

Comment:
Or we can also not log them at all, and just duplicate the audio "time delay" information ?